frontend/model/notifications/templates.js

```diff
@@ -93,9 +108,13 @@ export default ({
   },
   INCOME_DETAILS_OLD (data: { months: number, lastUpdatedDate: string }) {
     return {
+      title: L('Update income details'),
       body: L("You haven't updated your income details in more than {months} months. Would you like to review them now?", {
         months: Math.floor(data.months) // Avoid displaying decimals
       }),
+      plaintextBody: L("You haven't updated your income details in more than {months} months. Would you like to review them now?", {
+        months: Math.floor(data.months) // Avoid displaying decimals
+      }),
```

Comment:
In the case where there's no difference, can we leave it `undefined` and default to `body`?